src/app/api/scrim/update-scrim-options/route.ts

```diff
@@ -11,6 +11,7 @@ type UpdateScrimBody = {
   scrimId: number;
   date: string;
   guestMode: boolean;
+  maps: [{ id: number; replayCode: string }];
```

Comment:
This type should be `{ id: number; replayCode: string }[]`, this is currently a tuple type and not an array type